test/integration/spec/connect.js

```diff
@@ -524,6 +524,9 @@ describe('connect', function() {
       }
       cancelablePromises = tokens.map(token => connect(token, options));
       rooms = await waitFor(cancelablePromises, 'connecting to rooms');
+      if (defaults.largeRoom && !withoutTracks) {
+        await Promise.all(rooms.map(({ localParticipant }) => tracksPublished(localParticipant, 2)));
```

Comment:
To make it easier on developers should we emit `trackPublished` even for non-large rooms after connect?

Reply:
@makarandp0 I don't think we should change the existing API contract for that reason. It will be deprecated in the future anyway.